xmlStyle.go

```diff
@@ -187,11 +188,12 @@ func (styles *xlsxStyleSheet) getStyle(styleIndex int) *Style {
 
 		if xf.FontId > -1 && xf.FontId < styles.Fonts.Count {
 			xfont := styles.Fonts.Font[xf.FontId]
-			style.Font.Size, _ = strconv.Atoi(xfont.Sz.Val)
+			style.Font.Size, _ = strconv.ParseFloat(xfont.Sz.Val, 64) // returns err
```

Comment:
I'm not a fan of this comment. The previous error just ignored the error if there was one.